test/test_distributions.py

```diff
@@ -2849,12 +2849,24 @@ def test_kl_expanded_normal(batch_shape, event_shape):
     assert_allclose(actual, expected)
 
 
-@pytest.mark.parametrize("batch_shape", [(), (1,), (2, 3)], ids=str)
-def test_kl_multivariate_normal_consistency_with_independent_normals(batch_shape):
-    event_shape = (5, )
-    shape = batch_shape + event_shape
+@pytest.mark.parametrize(
+    "batch_shape_p, batch_shape_q",
+    [
+        ((), ()),
+        ((1,), (1,)),
+        ((2, 3), (2, 3)),
+        ((5, 2, 3), (2, 3)),
```

Comment:
could you change this to `(5, 1, 3)` and `(2, 3)`?

Reply:
done